packages/compile/src/_shared/helpers.js

```diff
@@ -200,15 +209,6 @@ export let isIterable = obj => {
   }
   return typeof obj[Symbol.iterator] === 'function'
 }
-export let stringToId = str => {
```

Comment:
This function was a holdover from the old web app generator, no longer used anywhere, so I deleted it.